web-server/v0.4/src/services/explore.js

```diff
@@ -19,3 +19,29 @@ export default async function querySharedSessions(params) {
     },
   });
 }
+
+export async function queryEditDescription(params) {
+  const { datastoreConfig, id, value } = params;
+
+  const endpoint = `${datastoreConfig.graphql}`;
+
+  return request.post(endpoint, {
+    data: {
+      query: `
+        mutation($description: String!,$id: ID!) {
+        updateUrl(
+          data: {description: $description}
+          where: {id: $id})
+        {
+          id
+          config
+          description
```

Comment:
Why do we need a separate method/function for fetching the description when the `querySharedSessions` method/function already returns the description?

Reply:
Or is this not query, but a write method?  If so, then we need to rename this from `query` to `write` or something to show/document the intended behavior.

Reply:
@portante we have been using the `query` keyword in every function which is used for modifying/fetching data in the `service` folder. Should I update the name of the method to `updateDescription` instead?

Reply:
Yes, that would help, and also add a comment that describes the intended behavior of the method/function.

Reply:
@portante @gurbirkalsi I have updated the name of the method to highlight the intended method but not sure if it is going with the naming conventions used in the dashboard. Please let me know if it needs to be changed.

Reply:
> Yes, that would help, and also add a comment that describes the intended behaviour of the method/function.

@portante Done!